Marlin/src/lcd/dogm/u8g_dev_tft_320x240_upscale_from_128x64.cpp

```diff
@@ -112,8 +112,8 @@ static uint32_t lcd_id = 0;
 #define U8G_ESC_DATA(x) (uint8_t)(x >> 8), (uint8_t)(x & 0xFF)
 
 static const uint8_t page_first_sequence[] = {
-  U8G_ESC_ADR(0), LCD_COLUMN, U8G_ESC_ADR(1), U8G_ESC_DATA(X_MIN), U8G_ESC_DATA(X_MAX),
-  U8G_ESC_ADR(0), LCD_ROW,    U8G_ESC_ADR(1), U8G_ESC_DATA(Y_MIN), U8G_ESC_DATA(Y_MAX),
+  U8G_ESC_ADR(0), LCD_COLUMN, U8G_ESC_ADR(1), U8G_ESC_DATA(X_LO), U8G_ESC_DATA(X_HI),
+  U8G_ESC_ADR(0), LCD_ROW,    U8G_ESC_ADR(1), U8G_ESC_DATA(Y_LO), U8G_ESC_DATA(Y_HI),
```

Comment:
Please revert that... unrelated

Reply:
Part of @thinkyhead 's cleanup commit so it'll be up to him.

Reply:
from what i see, seems from you : https://github.com/MarlinFirmware/Marlin/pull/14044/commits/5d344ae6e95d6f279cad17329c254f5a6bf321ba